docs/cmd/kn_service_update.md

```diff
@@ -40,6 +41,7 @@ kn service update NAME [flags]
   -p, --port int32               The port where application listens on.
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
+      --wait-timeout int         Seconds to wait before giving up on waiting for service to be ready. (default 60)
```

Comment:
shouldnt we set wait-timeout to be time.Duration. otherwise what's the unit of this flag?

Reply:
As the message says, its "Seconds to wait". I think (hope) that's good enough, but we can improve on adding a unit, too (like moving to a string and parsing "1min"). personally, I think that's overkill.